tests/acceptance/sync-past-orders/SyncPastOrdersCest.php

```diff
@@ -85,40 +82,6 @@ public function testNoButtonDisplayedWhenIntegrationDisabled(AcceptanceTester $I
 		$I->dontSeeElementInDOM('a#ckwc_sync_past_orders');
 	}
 
-	/**
-	 * Test that no button is displayed on the Integration Settings screen
-	 * when the Integration is enabled, and no API credentials are specified.
-	 *
-	 * @since   1.4.3
-	 *
-	 * @param   AcceptanceTester $I  Tester.
-	 */
-	public function testNoButtonDisplayedWhenIntegrationEnabledWithNoAPICredentials(AcceptanceTester $I)
```

Comment:
This test is removed as we now only display a `Connect` button when no access and refresh token exist.